src/usr/local/sbin/ovpn_auth_verify

```diff
@@ -22,7 +22,11 @@
 
 
 if [ "$1" = "tls" ]; then
-	RESULT=$(/usr/local/sbin/fcgicli -f /etc/inc/openvpn.tls-verify.php -d "servercn=$2&depth=$3&certdepth=$4&certsubject=$5")
+	# Depth in the certificate chain where the cert to verify is.
+	# "0" is the usual value here, where the client certificate is
+	check_depth=0
```

Comment:
Should `check_depth` always be `0` here? Or should it be what the user selected for depth in the OpenVPN server?

Reply:
done